cfgov/unprocessed/apps/tccp/css/main.less

```diff
@@ -23,6 +23,14 @@
   }
 }
 
+// Some cards list a lot of available locations and disrupt the table layout.
+// Cap the width of the availability cell on non-mobile screens to prevent this.
+@media only screen and (min-width: @bp-sm-min) {
+  td[data-label='Availability'] {
```

Comment:
Note to self: if we ever change this label to something else, we'll need to update it here as well.